ruff.toml

```diff
@@ -0,0 +1,3 @@
+line-length = 120
+target-version = "py311"
+indent-width = 4
```

Comment:
It's somewhat ironic that the ruff config file is poorly formatted: it misses an ending newline :)

Reply:
fixed and removed the indent-width since it's the default value